Jenkinsfile

```diff
@@ -0,0 +1,497 @@
+                                        "compose_rev='N/A'\n" +
+                                        "branch=${branch}\n" +
+                                        "ref=fedora/${branch}/${basearch}/atomic-host\n" +
+                                        "rev=${fed_rev}\n" +
+                                        "repo=${fed_repo}\n" +
+                                        "namespace=${namespace}\n" +
+                                        "username=fedora-atomic\n" +
+                                        "test_guidance=''\n" +
+                                        "status=success",
+
+                                messageType: 'Custom',
+                                overrides: [topic: "${topic}"],
+                                providerName: 'fedora-fedmsg'
+
+                        // Provision resources
+                        env.DUFFY_OP = "--allocate"
+                        allocDuffy("${current_stage}")
+
+                        echo "Duffy Allocate ran for stage ${current_stage} with option ${env.DUFFY_OP}\r\n" +
+                                "ORIGIN_WORKSPACE=${env.ORIGIN_WORKSPACE}\r\n" +
+                                "ORIGIN_BUILD_TAG=${env.ORIGIN_BUILD_TAG}\r\n" +
+                                "ORIGIN_CLASS=${env.ORIGIN_CLASS}"
+                        def job_props = "${env.ORIGIN_WORKSPACE}/job.props"
+                        def job_props_groovy = "${env.ORIGIN_WORKSPACE}/job.groovy"
+                        convertProps(job_props, job_props_groovy)
+                        load(job_props_groovy)
+
+                        // Stage resources - ostree compose
+                        setupStage()
+
+                        // Rsync Data
+                        writeFile file: "${env.ORIGIN_WORKSPACE}/task.env",
+                                  text: "export BUILD=\"${fed_branch}\"\n" +
+                                        "export JENKINS_JOB_NAME=\"${JOB_NAME}-${current_stage}\"\n" +
+                                        "export JENKINS_BUILD_TAG=\"${BUILD_TAG}-${current_stage}\"\n" +
+                                        "export OSTREE_BRANCH=\"\${OSTREE_BRANCH:-}\"\n"
+                        rsyncResults("$current_stage")
+
+                        def ostree_props = "${env.ORIGIN_WORKSPACE}/logs/ostree.props"
+                        def ostree_props_groovy = "${env.ORIGIN_WORKSPACE}/ostree.props.groovy"
+                        convertProps(ostree_props, ostree_props_groovy)
+                        load(ostree_props_groovy)
+
+
+                        // Check if a new ostree image compose is needed
+                        checkLastImage()
+                        if (fileExists("${env.WORKSPACE}/NeedNewImage.txt")) {
+                            stage("ci-pipeline-ostree-image-compose") {
+                                current_stage = "ci-pipeline-ostree-image-compose"
```

Comment:
@p3ck this is updated in the latest patch.  I also did fill in rev and repo to use fed_rev and fed_repo in the message